manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -90,12 +100,12 @@ func (r *FybrikApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	if os.Getenv("ENABLE_WEBHOOKS") != "true" && (string(applicationContext.Status.ValidApplication) == "" || observedStatus.ValidatedGeneration != appVersion) {
 		// do validation on applicationContext
 		err := applicationContext.ValidateFybrikApplication(ApplicationTaxonomy)
-		log.V(0).Info("Reconciler validating Fybrik application")
+		log.Trace().Msg("Reconciler validating Fybrik application")
```

Comment:
Either debug, or could be removed completely.

Reply:
OK